libraries/aws_mq_configuration.rb

```diff
@@ -4,37 +4,37 @@
 
 class AWSMQConfiguration < AwsResourceBase
   name 'aws_mq_configuration'
-  desc 'Describes a Amazon MQ configuration.'
-
-  example "
-    describe aws_mq_configuration(configuration_id: 'test1') do
+  desc 'Returns information about the specified configuration.'
+  example <<-EXAMPLE
+    describe aws_mq_configuration(configuration_id: 'CONFIGURATION_ID') do
       it { should exist }
     end
-  "
+  EXAMPLE
 
   def initialize(opts = {})
-    opts = { configuration_id: opts } if opts.is_a?(String)
     super(opts)
-    validate_parameters(required: [:configuration_id])
-
-    raise ArgumentError, "#{@__resource_name__}: configuration_id must be provided" unless opts[:configuration_id] && !opts[:configuration_id].empty?
+    validate_parameters(required: %i(configuration_id))
+    raise ArgumentError, "#{@__resource_name__}: configuration_id must be provided" if opts[:configuration_id].blank?
     @display_name = opts[:configuration_id]
-    resp = @aws.mq_client.describe_configuration({ configuration_id: opts[:configuration_id] })
-    @configurations = resp.to_h
-    create_resource_methods(@configurations)
-  end
-
-  def configuration_id
-    return nil unless exists?
-    @configurations[:configuration_id]
+    catch_aws_errors do
+      resp = @aws.mq_client.describe_configuration({ configuration_id: opts[:configuration_id] })
+      @res = resp.to_h
```

Comment:
`@configurations` is a better variable name than `@res`

Reply:
Changes done